src/Wallabag/CoreBundle/Command/InstallCommand.php

```diff
@@ -223,6 +223,14 @@ protected function setupAdmin()
         $config->setItemsPerPage($this->getContainer()->getParameter('wallabag_core.items_on_page'));
         $config->setRssLimit($this->getContainer()->getParameter('wallabag_core.rss_limit'));
         $config->setLanguage($this->getContainer()->getParameter('wallabag_core.language'));
+        $config->setCarrot(false);
+        $config->setShareDiaspora(false);
+        $config->setDiasporaUrl('http://diasporapod.com');
+        $config->setShareShaarli(false);
+        $config->setShaarliUrl('http://myshaarli.com');
+        $config->setShareMail(true);
+        $config->setShareTwitter(true);
```

Comment:
Instead of `Share*`, I would rather write them `Enable*`  instead. What do you think?